hw2/04-background.js

```diff
@@ -0,0 +1,59 @@
+// Add your code here
+window.addEventListener('load', (event) => {
+  if (!nIntervId) {
+    nIntervId = setInterval(changeBackground, 3000);
+  }
+});
+
+function getRGBA() {
+  const r = Math.floor(Math.random() * 256);
+  const g = Math.floor(Math.random() * 256);
+  const b = Math.floor(Math.random() * 256);
```

Comment:
I may be wrong, but I think it should be 255. 0 to 255. as the floor will include 256 as an option.

Reply:
I think Math.random will give a number between the range [0,256)